src/pages/specs/Details.spec.js

```diff
@@ -0,0 +1,17 @@
+import { create } from 'react-test-renderer';
+import Details from '../Details';
+
+
+jest.mock('react-router-dom', () => ({ Link: 'Link' }));
+jest.mock('../styledComponents', () => ({ Card: 'Card', RocketDetails: 'RocketDetails', RocketName: 'RocketName' }));
+jest.mock('../../hooks/useLaunchList', () => () => ({
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/megwali/spacex/pullRequest?prid=7693722)
```suggestion
jest.mock("../../hooks/useLaunchList", () => () => ({
```